src/locale/uk.js

```diff
@@ -1,6 +1,14 @@
 // Ukrainian [uk]
 import dayjs from 'dayjs'
 
+const monthFormat = 'січня_лютого_березня_квітня_травня_червня_липня_серпня_вересня_жовтня_листопада_грудня'.split('_')
+const monthStandalone = 'січень_лютий_березень_квітень_травень_червень_липень_серпень_вересень_жовтень_листопад_грудень'.split('_')
+
+const monthShortFormat = 'сiч_лют_бер_квiт_трав_черв_лип_серп_вер_жовт_лист_груд'.split('_')
```

Comment:
there's no difference in `monthShort`, right?

Reply:
yes, you're right

Reply:
why not using the original `monthsShort`?

Reply:
for the same logic with long names
(I looked Russian locale file)

Reply:
There's no need actually.
You can just add `monthFormat ` and `monthStandalone `

Reply:
ok